src/confdb/data/Path.java

```diff
@@ -152,13 +152,13 @@ public boolean setType(Type type) {
 
 	/**
 	 * a path is a valid output path of a stream if
-	 * 1) it is a final path
+	 * 1) it is a EndPath or FinalPath
 	 * 2) it contains exactly one entry
 	 * 3) that entry is that streams output module
 	 * if no stream is specified then it just looks for one output module to be present
 	 */
 	public boolean isOutputPathOfStream(Stream stream){
```

Comment:
So we need to decide if we allow now other modules to be on the EndPath, such as a prescaler. Right now final paths are forbidden by the framework to have anything one them and this function enforces that in the GUI. 

Currently  with a prescaler in an output path, you would then get  an error in config checks that the path exists but is no longer valid.

Reply:
(I had this in mind for a follow-up PR, to avoid slowing this one down.)

I pushed a small update improving the check in
https://github.com/cms-sw/hlt-confdb/blob/3a111c2aed5d0a86e3fab04888fc81def40266d0/src/confdb/data/Path.java#L174

This should allow users to manually add modules to the `EndPath` (while the requirement of only-one module remains for FinalPaths). I did not go as far as adding by default a prescale module to the EndPath "stream paths", and I would leave this (if needed) to a follow-up PR.